dataactbroker/handlers/accountHandler.py

```diff
@@ -137,6 +144,117 @@ def login(self,session):
         return self.response
 
 
+    def max_login(self,session):
+        """
+
+        Logs a user in if their password matches
+
+        arguments:
+
+        session  -- (Session) object from flask
+
+        return the reponse object
+
+        """
+        try:
+            safeDictionary = RequestDictionary(self.request)
```

Comment:
We need this to process the POST contents from the front-end. It's not the response we're getting directly from MAX. The front-end does the initial request, gets the payload, then forwards it to the broker.